modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaAddAddressMethodCheck.java

```diff
@@ -0,0 +1,144 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+import com.liferay.source.formatter.parser.JavaClass;
+import com.liferay.source.formatter.parser.JavaClassParser;
+import com.liferay.source.formatter.parser.JavaMethod;
+import com.liferay.source.formatter.parser.JavaTerm;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaAddAddressMethodCheck extends BaseUpgradeCheck {
+
+	@Override
+	protected String format(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		String newContent = content;
+
+		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, content);
+
+		for (JavaTerm childJavaTerm : javaClass.getChildJavaTerms()) {
+			if (!childJavaTerm.isJavaMethod()) {
+				continue;
+			}
+
+			JavaMethod javaMethod = (JavaMethod)childJavaTerm;
+
+			String javaMethodContent = javaMethod.getContent();
+
+			Matcher matcher = _addAdressPattern.matcher(javaMethodContent);
+
+			while (matcher.find()) {
+				String methodCall = JavaSourceUtil.getMethodCall(
+					javaMethodContent, matcher.start());
+
+				String variableName = getVariableName(methodCall);
+
+				if (!variableName.contains("AddressLocalServiceUtil") &&
+					!variableName.contains("AddressServiceUtil") &&
+					!hasClassOrVariableName(
+						"AddressLocalService", content, content, methodCall) &&
+					!hasClassOrVariableName(
+						"AddressService", content, content, methodCall)) {
+
+					continue;
+				}
+
+				String message = StringBundler.concat(
+					"Unable to format method addAddress from ",
+					"AddressLocalServiceUtil, AddressServiceUtil, ",
+					"AddressLocalService and AddressService. Fill the new ",
```

Comment:
@TamyrisTorres, can you sort alphabetically?

AddressLocalService, AddressLocalServiceUtil, AddressService, AddressServiceUtil